specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.generatedetailedcostreport.json

```diff
@@ -0,0 +1,373 @@
+          "type": "string",
+          "enum": [
+            "ActualCost",
+            "AmortizedCost"
+          ],
+          "x-ms-enum": {
+            "name": "GenerateDetailedCostReportMetricType",
+            "modelAsString": true
+          }
+        },
+        "timePeriod": {
+          "description": "Has time period for pulling data for the cost detailed report. Can only have one of either timePeriod or invoiceId or billingPeriod parameters. If none provided current month cost is provided.",
+          "$ref": "#/definitions/GenerateDetailedCostReportTimePeriod"
+        },
+        "billingPeriod": {
+          "description": "Billing Period in YearMonth(e.g. 202008) format. Only for legacy enterprise customers can use this. Can only have one of either timePeriod or invoiceId or billingPeriod parameters. If none provided current month cost is provided.",
+          "type": "string"
+        },
+        "invoiceId": {
+          "description": "Invoice Id for PayAsYouGo customers and Modern billing profile scope. Can only have one of either timePeriod or invoiceId or billingPeriod parameters. If none provided current month cost is provided.",
+          "type": "string"
+        },
+        "customerId": {
+          "description": "Customer Id for Modern (Invoice Id and billing profile is also required for this).",
+          "type": "string"
+        }
+      }
+    },
+    "GenerateDetailedCostReportTimePeriod": {
+      "description": "The start and end date for pulling data for the cost detailed report.",
+      "properties": {
+        "start": {
+          "description": "The start date to pull data from. example format 2020-03-15",
+          "type": "string"
+        },
+        "end": {
+          "description": "The end date to pull data to. example format 2020-03-15",
+          "type": "string"
+        }
+      },
+      "required": [
+        "start",
+        "end"
+      ]
+    },
+    "GenerateDetailedCostReportOperationStatuses": {
+      "description": "The status of the long running operation for cost detailed report.",
+      "properties": {
+        "id": {
+          "description": "The id of the long running operation.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the long running operation.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The status of the long running operation.",
+          "$ref": "#/definitions/Status"
+        },
+        "type": {
+          "description": "The type of the long running operation.",
+          "type": "string"
+        },
+        "error": {
+          "description": "The details of the error.",
+          "$ref": "common-types.json#/definitions/ErrorDetails"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "The properties of the resource generated.",
```

Comment:
>the resource generated. [](http://example.com/codeflow?start=44&length=23)

What do we mean by 'the resource generated'? Can you clarify what the resource is,in this context?

Reply:
@absyed please comment